### PR TITLE
Confirmation order message goes under the search dialog result (GH3066)

### PR DESCRIPTION
**Pull request by reyhanehsalkhi**
closes GH-3066

### CODE REVIEW

File: projects/storefrontstyles/scss/components/checkout/order-confirmation/_order-confirmation-thank-you-message.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 @import '../../../theme';
 
 %cx-order-confirmation-thank-you-message {
+
+  #cx-thank-you-header {
```
Comment (LTiger14) [inline projects/storefrontstyles/scss/components/checkout/order-confirmation/_order-confirmation-thank-you-message.scss]: This should only be in calydon file

File: projects/storefrontstyles/scss/themes/calydon/components/checkout/order-confirmation/_index.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+@import './order-confirmation-thank-you-message';
```
Comment (LTiger14) [inline projects/storefrontstyles/scss/themes/calydon/components/checkout/order-confirmation/_index.scss]: Fix indentation and add trailing line

File: projects/storefrontstyles/scss/themes/calydon/components/checkout/order-confirmation/_order-confirmation-thank-you-message.scss [hunk 3]
```diff
@@ -1,7 +1,5 @@
 %cx-order-confirmation-thank-you-message-calydon {
-
   #cx-thank-you-header {
```
Comment (developpeurweb) [inline projects/storefrontstyles/scss/themes/calydon/components/checkout/order-confirmation/_order-confirmation-thank-you-message.scss]: @LTiger14 @reyhanehsalkhi 
I came across this PR while reviewing the Praetorian's board, sorry to cut in, but just wanted to comment on the fact that we're not using css ID selectors anywhere in the Design System, so i would suggest changing this selector to a class selector like `.cx-thank-you-header` ... the sass placeholder makes it specific enough already anyway.

File: projects/storefrontlib/src/cms-components/order-confirmation/components/order-confirmation-thank-you-message/order-confirmation-thank-you-message.component.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="order$ | async as order">
-  <header class="cx-page-header">
+  <header class="cx-page-header" id="cx-thank-you-header">
```
Comment (plabadie) [inline projects/storefrontlib/src/cms-components/order-confirmation/components/order-confirmation-thank-you-message/order-confirmation-thank-you-message.component.html]: I believe that changes in the html DOM is considered a breaking change.  Please use a feature level flag with the version number in which this fix will be released in.
https://sap.github.io/cloud-commerce-spartacus-storefront-docs/feature-flags-and-code-deprecation

**Comment by plabadie** [comment]
This is just a small convention thing: please use a dash in the issue numver in the branch name,
like: `feature/GH-3066`

**Pull request closed without merging**